### ISSUE TITLE
Bug: Contributor chart not loading

### ISSUE THREAD

**Issue by BekahHW** [open-sauced/app#3650]
### Describe the bug

When I look at the 30day view for gatsby, the contributors aren't there. 

![image](https://github.com/open-sauced/app/assets/34313413/c8d99fb9-828e-46d0-81d0-15478224d15c)

### Steps to reproduce

1. Go to https://app.opensauced.pizza/s/gatsbyjs/gatsby?range=30
2. See an empty chart.

**Comment by nickytonline** [comment]
Are there any errors in the browser devtools console, or any network errors in the network panel?

**Comment by Arun-cn** [comment]
yes, load resource: the server responded with a status of 429 Too Many Requests

**Comment by BekahHW** [comment]
@nickytonline Happening with [Reactotron](https://app.opensauced.pizza/s/infinitered/reactotron?range=90) today. I'm not getting any errors. My suspicion was that once the number falls below a certain point, the graph isn't generated. Once I go to 90 days, it works. However, when I looked at [our intro repo](https://app.opensauced.pizza/s/open-sauced/intro?range=30), we had 10 contributors -- the same as reactotron -- and it generates.

<img width="618" alt="image" src="https://github.com/open-sauced/app/assets/34313413/8c44af39-1262-49e2-833a-c1e2245346ff">
